sale_exception/views/sale_view.xml

```diff
@@ -69,6 +69,7 @@
             <field name="name">sale_exception.view_order_form</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_order_form"/>
+            <field name="groups_id" eval="[(4, ref('base.group_sale_salesman'))]"/>
```

Comment:
Why this one only on sales manager?

Reply:
I think you are mistaken.

Reply:
Agh, incorrect line for commenting. It's in the above change the one that it's only on managers.